tensorflow/core/graph/mkl_graph_util.h

```diff
@@ -104,12 +104,20 @@ static const char* kMklQuantizedOpLabelPattern = "label='QuantizedMklOp'";
 
 // Prefix that we add to Tensorflow op name to construct Mkl op name.
 static const char* const kMklOpPrefix = "_Mkl";
+static const char* const kMklEagerOpPrefix = "_MklEager";
```

Comment:
Can we add `eager_mode` (or `is_eager`) as an op attribute instead? This way we don't need to rename the op just to pass `eager_mode` through template parameter.

Reply:
@penpornk I have put a comment right before the declaration of "_MklEager", for refactoring purpose in near future (original developer is on a long vacation). I have included your suggestion in the comment.

Reply:
I guess that the developer (Mahmoud) wanted to rename the op in order to make sure that the renaming only occurs in pure eager mode  (versus non-eager-mode, all mkl op rewriting is done  thru MKL layout pass (graph/mkl_layout_pass) which is only triggered at the state of
POST_PARTITIONING (which never reach in pure eager mode!).

So, for now, I would recommend that we do refactoring one the original developer is back)
Thanks

Reply:
But why exactly do we need to rename the op, though? Making `eager_mode` an attribute is enough to tell the difference, and is much cleaner (no need to spawn new `_MklEager*` ops every time we add a rewrite rule).

Reply:
done with change